### DISCUSSION TITLE
PiPedal v1.3.53 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#205] [Releases]
Major features:
- Snapshots. For a discussion of what snapshot are, see [An Intro to Snapshots](https://rerdavies.github.io/pipedal/Snaphots.html)

- New <b>Performance View</i></b> providing an user-interface optimized for live performance rather than editing of plugins.

- New System Midi Bindings for snapshot selection, and for previous and next bank.

This should have probably gone out as a Beta release, but the fixes for the "Audio Stopped" error are significant. I would recommend updating to this release as soon as you can. Please consider the snapshot user interface to be somehwat fluid for now. I am open to suggestions and feedback on the new feature.

Bug Fixes:
- A fix for the dreaded "Audio Stopped" error (and associated memory corruption). You may underrun, but at least audio won't stop.

- Further improvements to clean (and timely) shutdown procedure.

- Minor UI improvements for adaptive display on small devices.

- A bug in TooB Freeverb that causes memory use to increase exponentially each time the plugin is loaded.

---
The .asc file is a gpg digital signature for the .deb file. You can verify the signature using the following command:

    gpg --verify pipedal_1.2.53_arm64.deb.asc pipedal_1.2.53_arm64.deb

Files are signed with the following public key:

&nbsp;&nbsp;&nbsp;&nbsp;[PiPedal Project &lt;rerdavies@gmail.com&gt; - 2D1F 39DB B1F8 1941 2B67  8F88 E9D7 081E 08E3 D85C ](https://rerdavies.github.io/pipedal/pipedal-public-key.asc)

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/'>PiPedal v1.3.53 Release</a>.</em>

**Comment by BorisSutin** [comment]
Great job Robin. The project is really becoming very cool ! 

Thank you very much.
